api-guide/overview.md

```diff
@@ -42,6 +47,9 @@ If you are new to the platform, see
 
 ## Sales units that do not have API access
 
+It is not possible for a merchants to use the Report API directly with Vippsnummer sales units,
+since there is no API access.
+
```

Comment:
```suggestion
It is not possible for merchants to use the Report API directly with Vippsnummer sales units,
since the sales unit doesn't have API access.

```

Reply:
Also, Vippsnummer vs vm number here?